frontend/src/components/beta/transactions/TransactionReceipt.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { TransactionReceipt as TxReceipt } from "../../../types/common";
```

Comment:
Are you sure we need to keep names short? `TransactionReceipt` is pretty readable and not that bloated as for me.

Reply:
Because component and type has the same name

Reply:
`TransactionReceiptView`?